vci-puppeteer/vci-approve-variants.js

```diff
@@ -307,6 +342,30 @@ function main() {
 	})();
 }
 
+function printRunStatistics() {
```

Comment:
This function is called after we iterate through every variant (so at the end of the variants list) or if the user is terminating the program.